### ISSUE TITLE
File /opt/wifi.sh not found, mismatching with the setup guide

### ISSUE THREAD

**Issue by rodaciri** [sipeed/LicheeRV-Nano-Build#72]
### Steps to Reproduce
1. Followed the instructions in the WiFi setup guide at [this section](https://wiki.sipeed.com/hardware/en/lichee/RV_Nano/5_peripheral.html#WIFI).
2. After writing the ssid and password into the "/etc/wpa_supplicant.conf" file, I tried to run the command `/opt/wifi.sh`.
3. But I got a "not found" message.
4. Then, doing `stat /opt/wifi.sh` I confirmed that file doesnt exist.
5. Finally I noticed the "/opt" directory is empty.

### Additional Information
- I tested it out with images:[20241129](https://github.com/sipeed/LicheeRV-Nano-Build/releases/tag/20241129) and [20241220](https://github.com/sipeed/LicheeRV-Nano-Build/releases/tag/20241220).

Please let me know if there is another way to set up the wifi. BTW for some reason internet connection via ETH is not working either.

**Comment by Neutree** [comment]
refer to this function did https://github.com/sipeed/MaixCDK/blob/04fa210e368ce0dfa3b7660b048bea41e94a3ebc/components/network/port/linux/maix_wifi.cpp#L445

**Comment by FreezyExp** [comment]
after editing /etc/wpa_supplicant.conf 
reboot

you can then ssh into the device over wifi

**Comment by lesept777** [comment]
Hi, I have the same problem and rebooting the device doesn't solve it.
@Neutree I don't understand your answer. What should I do with it ?

**Comment by lesept777** [comment]
I created the files /boot/wifi.ssid and /boot/wifi.pass
I executed `/etc/init.d/S30wifi  restart `but not sure it worked. Here is what I get, and the console seems frozen

> kill: you need to specify whom to kill
> kill: you need to specify whom to kill
> Your kernel has module support but you don't have modinfo installed.
> It is highly recommended to install modinfo (typically from kmod).
> Warning: driver detection without modinfo may yield inaccurate results.
> 
> Requested device "wlan0mon" does not exist.
> Run /usr/sbin/airmon-ng without any arguments to see available interfaces
> wifi mode: sta
> ssid: lesept
> wifi.pass: ***
> Successfully initialized wpa_supplicant
> nl80211: kernel reports: Registration to specific type not supported
> # udhcpc: started, v1.36.1
> udhcpc: broadcasting discover
> udhcpc: broadcasting discover
> udhcpc: broadcasting discover
> udhcpc: broadcasting discover
> udhcpc: broadcasting discover
> udhcpc: broadcasting discover
> udhcpc: broadcasting discover
> udhcpc: broadcasting discover
> udhcpc: broadcasting discover
> udhcpc: broadcasting discover
> udhcpc: no lease, forking to background
> 

Not sure what to do next...
